recipes/smudge

```diff
@@ -0,0 +1,5 @@
+(smudge
+	:fetcher github
+	:repo "danielfm/smudge"
+	:branch "master"
+	:files ("*.el"))
```

Comment:
Default recipe should work here (and gives you room to expand), and we only require you to specify `:branch` if you aren't using your repo's default branch -- and in any case MELPA prefers to use the default branch when possible.
```suggestion
	:repo "danielfm/smudge")
```